components/utils/scripts/events.js

```diff
@@ -415,9 +415,9 @@ elation.extend("events", {
     var newev = {};
     for (let i = 0; i < this.cloneattrs.length; i++) {
       let attr = this.cloneattrs[i];
-      let foo = elation.utils.any(overrides[attr], ev[attr]);
```

Comment:
so the `overrides[attr]` did not exist..threw an error..and halted further room-loading.